config/test.js

```diff
@@ -10,4 +10,9 @@ module.exports = {
   enableRequestID: false,
 
   mozillaUserId: 1337,
+
+  // Disable all experiments by default.
+  experiments: {
+    '20210531_amo_download_funnel_experiment': false,
```

Comment:
Can we import `config/default.js` and spread `config.experiments` here instead? (if we really want to disable all experiments by default)

Reply:
I'm not sure I follow. Do you mean to get the keys from `config/default.js` and explicitly set each one to `false`?

Reply:
Yes:

```js
...Object.keys(defaultConfig.experiments).reduce((acc, key) => {
  return { ...acc, [key]: false };
}, {});
```

Reply:
Right, I thought that might have been what you meant. Do you think that is a better approach, or the mocking approach that is now present in this patch? Or maybe something else?